common/src/cobs/shared.rs

```diff
@@ -134,11 +135,26 @@ pub type Replies = Vec<Comment>;
 pub struct Comment<R = ()> {
     pub author: Author,
     pub body: String,
+    #[serde(serialize_with = "serialize_reactions")]
```

Comment:
I think this would be more easily achieved with `#[serde(transparent)]` on the `Reaction` struct, if I'm not mistaken.

Reply:
Nice good catch, did reduce the PR to just that. 😅